Express 102 lesson: add additional resource link (#23411)
TheOdinProject/curriculum: nodeJS/express-basics/Express-Lesson-2.md

This section contains helpful links to other content. It isn't required, so consider it supplemental.

- For a deeper explanation of MVC you could read [this article from freeCodeCamp](https://medium.freecodecamp.org/simplified-explanation-to-mvc-5d307796df30).

> - This [crash course video](https://www.youtube.com/watch?v=DZBGEVgL2eE) from Web Dev Simplified gives a quick run-down of how you expect to use MongoDB (Mongoose) alongside Node.js, as well as some of the advanced things you can do related with object schemas

Reviewer: I'm not sure I'd call a 33 minute video "quick"

```suggestion
- This [crash course video](https://www.youtube.com/watch?v=DZBGEVgL2eE) from Web Dev Simplified gives a run-down of how you should expect to use MongoDB (Mongoose) in Node.js, as well as some advanced things you can do with object schemas.
```

Author: The first 20 minutes of the video goes through the basics of using Mongoose, and the rest is dedicated to the advanced stuff, so I'd reckon you could just skip to parts of the video that you needed to review/learn as a reference - that's was my intention of using the word "quick" here. Thanks for your suggestion btw 🙂
